fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/indexes/StandardIndexMaintainer.java

```diff
@@ -152,6 +173,12 @@ protected IndexEntry unpackKeyValue(@Nonnull final Subspace subspace, @Nonnull f
         return new IndexEntry(state.index, unpackKey(subspace, kv), decodeValue(kv.getValue()));
     }
 
+    @Nonnull
+    protected FDBIndexedRawRecord unpackIndexPrefetchRecord(@Nonnull final Index index, @Nonnull MappedKeyValue indexKeyValue) {
```

Comment:
Does this need to pass the `Index` as an argument, or can the method access `state.index` to retrieve that instead?

Reply:
Done